recipes/markitdown/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "markitdown" %}
+{% set version = "0.0.1a2" %}
```

This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md 

```suggestion
{% set version = "0.0.1a2" %}
```